modules/nw-ovn-kubernetes-session-affinity.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+:_content-type: CONCEPT
+[id="nw-ovn-kubernetes-session-affinity_{context}"]
+= Session affinity
+Session affinity is a feature that applies to Kubernetes `services` objects. You can use `session affinity` if you want to ensure that each time you connect to a ServiceVIP:Port the traffic is always load balanced to the same backend. For more information, including how to set session affinity based on a client's IP address, refer to link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity[Session affinity].
```

Comment:
* The object is called `Service` singular, capitalized. You can use our API reference docs or the "oc api-resources" command to look for common API object names.
* If you want to emphasize or define a term (session affinity in this case), then use italics
* "back end" two words when it's a noun (per ISG)
* Per ISG, "see" is preferred instead of "refer to"
* Are "ServiceVIP:Port" replaceable values that indicate something the user would replace (or depends on their system)? If so, then I'd use our normal replaceable value syntax
* Added comma

```suggestion
Session affinity is a feature that applies to Kubernetes `Service` objects. You can use _session affinity_ if you want to ensure that each time you connect to a <service_vip>:<port>, the traffic is always load balanced to the same back end. For more information, including how to set session affinity based on a client's IP address, see link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity[Session affinity].
```